module/Database/view/database/prospective-member/show.phtml

```diff
@@ -101,71 +101,77 @@ use Laminas\View\Renderer\PhpRenderer;
         </div>
     <?php endforeach; ?>
 
-    <?php
-    $form->prepare();
+    <?php if (isset($form)): ?>
```

Comment:
Move this into a separate `<div class="row"></div>` and extend the left column with a `CheckoutSession` history.

Reply:
Changed